website/content/docs/connect/gateways/mesh-gateway/wan-federation-via-mesh-gateways.mdx

```diff
@@ -7,12 +7,10 @@ description: |-
 
 # WAN Federation via Mesh Gateways
 
--> **1.8.0+:** This feature is available in Consul versions 1.8.0 and higher
-
 ~> This topic requires familiarity with [mesh gateways](/docs/connect/gateways/mesh-gateway/service-to-service-traffic-datacenters).
 
-WAN federation via mesh gateways allows for Consul servers in different datacenters
-to be federated exclusively through mesh gateways.
+Cross-DC control plane communication, also known as WAN federation,
+allows for Consul servers in different datacenters to be federated exclusively through mesh gateways.
```

Comment:
```suggestion
This topic describes how to enable cross-datacenter communication, also referred to as WAN federation, in the  control plane. WAN federation allows you to designate a primary datacenter and one or more secondary datacenters. In this configuration, the Consul servers in different datacenters are connected exclusively through mesh gateways.

## Background
```
I wanted to do a few things here: 

1. Use language that specifically calls out what the reader will get out of the topic (i.e. "This topic describes . . .")
2. Use the spelled-out "datacenter" term (as opposed to the abbreviation "DC") so that we don't confuse readers.
3. Try to include a brief definition of what we mean by "federation".    
4. Create a subheading for the following information. I'm not sure if "Background" is the best term. I would consider moving this information into the "Requirements" section with its own subheading.

To me, the graphics are a bit overwhelming, especially upfront. Would it make more sense to move them to the "Architecture" section?

Reply:
I tried to make a more concise version of the suggested first three sentences (that open the topic). Let me know what you think.

```
This topic describes how to configure Consul servers in federated datacenters to
communicate exclusively through mesh gateways.
```

Reply:
> To me, the graphics are a bit overwhelming, especially upfront. Would it make more sense to move them to the "Architecture" section?

I'll play with options for this, but I might want to address those in a separate PR if we don't come to a solution we like quickly. I don't want that to block this change to list requirements (which I want to use as a template for similar "requirements" content on other pages).

Reply:
And I haven't thought of a better subheading title than "Background", so I've left it as that for now.

Reply:
I think the images belong in the Background section because they directly pair with that text (which talks about the challenges of traditional WAN federation and how mesh gateways help).

One thing we could do is move these images to the bottom of the section and have a "before/after" style tabset. I'm not sure I like this more than the current approach, but wanted to present it for feedback:

![image](https://user-images.githubusercontent.com/85913323/155228216-68d5796e-577d-4ad0-9de1-bf5caf99d521.png)